.eslintrc.json

```diff
@@ -42,7 +42,8 @@
       { "blankLine": "always", "prev": "*", "next": "for" }
     ],
     "@typescript-eslint/ban-ts-comment": "off",
-    "react/display-name": "off"
+    "react/display-name": "off",
+    "@typescript-eslint/no-empty-function": "off"
```

Comment:
?

Reply:
W testach daje jakiemuś komponentowi ()=>{} jak nie potrzebuję danej funkcji i mi to wywalał eslint